astropy/modeling/tests/test_functional_models_quantities.py

```diff
@@ -0,0 +1,244 @@
+from collections import OrderedDict
+
+import pytest
+import numpy as np
+
+from ... import units as u
+from ...tests.helper import assert_quantity_allclose
+
+from ..functional_models import (Gaussian1D, GaussianAbsorption1D,
+                                 Sersic1D, Sine1D, Linear1D,
+                                 Lorentz1D, Voigt1D, Const1D,
+                                 Box1D, Trapezoid1D, MexicanHat1D,
+                                 Moffat1D, Gaussian2D, Const2D, Ellipse2D,
+                                 Disk2D, Ring2D, Box2D, TrapezoidDisk2D,
+                                 MexicanHat2D, AiryDisk2D, Moffat2D, Sersic2D)
+from ..fitting import LevMarLSQFitter
+
+try:
+    from scipy import optimize
+    HAS_SCIPY = True
+except ImportError:
+    HAS_SCIPY = False
+
+# TODO: GaussianAbsorption1D doesn't work with units because the 1- part doesn't
```

Comment:
Good question. `GaussianAbsorption1D` as it is now is only useful as a *multiplicative* factor to a continuum to create an absorption line. Otherwise, you can create absorption line without it by using `continuum_model - Gaussian1D`. (This model might had been written a long time ago before the math operators worked properly?) I wonder if `GaussianAbsorption1D` is used by anyone at all. Maybe deprecate it for future removal? That way, we can just say "this model is deprecated and does not support units".

Related reading: http://synphot.readthedocs.io/en/latest/synphot/spectrum.html#gaussian-absorption